tests/src/Kernel/fixtures/markup_rendering_patterns/inpage_navigation.yml

```diff
@@ -73,3 +73,36 @@ inpage_navigation_with_no_links:
       class: Drupal\Tests\oe_bootstrap_theme\PatternAssertion\InPageNavigationAssert
       expected:
         title: 'Page content'
+inpage_navigation_with_dropdown_id_and_id:
```

Comment:
dont think this is actually needed, nice the things are not new pattern fields, there is no actual new scenario, just add check to the assertBaseElements and we're fine

Reply:
That's correct for the base elements, but I would keep the scenario, just use custom values for both of them to see that the value passed is respected.